### ISSUE TITLE
wide-resnet N=6 not equivariant

### ISSUE THREAD

**Issue by PengzhenChan** [QUVA-Lab/escnn#101]
I was using the wide-resnet to test the equivariance of rotations of 60 degree, and I set the parameter 'N'=6. to test the equivariance, i transformed the input like this
`elements = m.gspace.testing_elements
    in_type = enn.FieldType(m.gspace, [m.gspace.trivial_repr] * 3)
    t = enn.GeometricTensor(x, in_type)
    x = torch.cat([t.transform(el).tensor for el in elements], dim=0)`
the resulted turned out to be equivariant when rotation is 180 degrees, but not equivariant under rotations of 60 and 120. I used the Maskmodule in the resnet to maintain the equivariance,but it was not effective. anything special  to pay attention to about this wide-resnet in your example? Appreciate your response!

**Comment by Gabri95** [comment]
Hi @PengzhenChan 

Unfortunately, it is impossible to achieve exact equivariance to rotations of images by 60 degrees, since this is not a perfect symmetry of the pixel grid. Equivariance to n=6 rotations can only be approximate. This discussion might be useful: https://github.com/QUVA-Lab/e2cnn/issues/61

Best,
Gabriele
